api/response/schema.py

```diff
@@ -0,0 +1,100 @@
+import graphene
+from graphene_sqlalchemy import SQLAlchemyObjectType
+from api.response.models import Response as ResponseModel
+from utilities.utility import validate_empty_fields, validate_rating_field
+from helpers.auth.user_details import get_user_from_db
+from graphql import GraphQLError
+from api.room.schema import Room
+from api.question.models import Question
+from helpers.auth.authentication import Auth
+
+
+class Response(SQLAlchemyObjectType):
+    class Meta:
+        model = ResponseModel
+
+
+class CreateRate(graphene.Mutation):
```

Comment:
This would be unnecessary if we had if had a question type and then in the response, restrict answers based on question.type

Reply:
I didn't create this, I will have a sync with @migot01, he created the mutation

Reply:
@shemogumbe I have implemented the requested changes